system/ee/legacy/libraries/Template.php

```diff
@@ -691,10 +691,11 @@ public function parse(&$str, $is_embed = false, $site_id = '', $is_layout = fals
      * @param  array $layout_vars Layout variables to parser, 'variable_name' => 'content'
      * @return string The parsed template/string
      */
-    private function parseLayoutVariables($str, $layout_vars)
+    public function parseLayoutVariables($str, $layout_vars)
     {
         $this->log_item("Layout Variables:", $layout_vars);
         $this->layout_conditionals = [];
+        $layout_conditionals = [];
```

Comment:
Line 790 attempts to assign a variable to the class property, but it's not created until line 725, or in my case not at all. Add this so it doesn't throw an error.